source/onboarding/team/index.html.md.erb

```diff
@@ -7,9 +7,32 @@ weight: 1
 
 # <%= current_page.data.title %>
 
-Slack -> create channels
+## Slack
 
-Config -> https://github.com/hmcts/cnp-jenkins-config/blob/master/team-config.yml
+Once you've received your `hmcts.net` email account you can self service sign-up to our slack workspace
+Visit [https://hmcts-reform.slack.com/](https://hmcts-reform.slack.com/) and click `create an account`
+
+After you've created your account please make sure to fill out your profile,
+set at least your full name, and in the what I do section put your team name and role.
```

Comment:
This is more relevant to onboarding people - but can see that we need to onboard at least 1 person before onboarding a team. Maybe need to duplicate for people as well when we come to that.

Reply:
Maybe people should come before team, with notes that if they are the first person in their team they should read the other docs?